<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="266" started-at="2017-01-15T15:28:44Z" finished-at="2017-01-15T15:28:44Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="266" started-at="2017-01-15T15:28:44Z" finished-at="2017-01-15T15:28:44Z">
      <class name="automationframework.UDF_TC">
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:automationframework.UDF_TC@1bce4f0a]" name="beforeMethod" is-config="true" duration-ms="214" started-at="2017-01-15T20:58:44Z" finished-at="2017-01-15T20:58:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="main()[pri:0, instance:automationframework.UDF_TC@1bce4f0a]" name="main" duration-ms="11" started-at="2017-01-15T20:58:44Z" finished-at="2017-01-15T20:58:44Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at turbopro_scenarious.HomePage.lnk_MyApplication(HomePage.java:16)
	at turbopro_module.Login_Action.Execute(Login_Action.java:18)
	at automationframework.UDF_TC.main(UDF_TC.java:45)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- main -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:automationframework.UDF_TC@1bce4f0a]" name="afterMethod" is-config="true" duration-ms="1" started-at="2017-01-15T20:58:44Z" finished-at="2017-01-15T20:58:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- automationframework.UDF_TC -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
